tests/unittests/tiglGetCrossSectionArea.cpp

```diff
@@ -0,0 +1,257 @@
+/*
+* Copyright (C) 2022 German Aerospace Center
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#include "test.h"
+#include "tigl.h"
+
+class GetCrossSectionAreaSimple : public ::testing::Test
+{
+protected:
+    static void SetUpTestCase()
+    {
+        const char* filename = "TestData/simpletest.cpacs.xml";
+        ReturnCode tixiRet;
+        TiglReturnCode tiglRet;
+
+        tiglHandle = -1;
+        tixiHandle = -1;
+
+        tixiRet = tixiOpenDocument(filename, &tixiHandle);
+        ASSERT_TRUE (tixiRet == SUCCESS);
+        tiglRet = tiglOpenCPACSConfiguration(tixiHandle, "", &tiglHandle);
+        ASSERT_TRUE(tiglRet == TIGL_SUCCESS);
+
+        // Add D150 test case
```

Comment:
No big deal, but I would probably have made this two separate test fixtures, one for the simple test and one for the D150.

Reply:
Do you think it would be a better style to create two seperate files for the test cases or rather arange both test cases in the same file but seperately? The second version would probably keep it more compact.

Reply:
If you *do* want to change this, I would keep both in one file. But to be honest, I have no compelling argument for having two seperate fixtures over having one as you implemented it, I just stumbled over it because I would have done it differently. You can either modularize the test based on functionality or use case. You went for functionality, which is fine.

So I suppose you can just leave it as it is.

Reply:
Ok, I understand. For now, I think I will then leave it as it is, but keep your remark in mind for similar situations in the future. Thanks a lot for your comments! I will push the changes regarding your comments in one minute.

Reply:
Just had to check something. Ok, I think it works.